task_functions.py

```diff
@@ -1321,3 +1324,32 @@ def put_object_minio(
             object_name=minio_path,
             file_path=local_path + filename,
         )
+
+
+def flush_cache(host, port, db, password):
+    redis_client = redis.Redis(
+        host=host,
+        port=port,
+        db=db,
+        password=password,
+    )
+    # Delete keys in the background in a different thread without blocking the server
+    flush_command = redis_client.execute_command("FLUSHALL ASYNC")
+    logging.info(f"Flush cache command status: {flush_command}")
+    if redis_client.keys():
+        raise Exception(f"****** Could not flush cache: {redis_client.keys()}")
+
+
+def execute_slow_requests():
+    session = requests.Session()
+    base_url = AIO_URL
+    slow_queries = ["q=rue", "q=rue%20de%20la", "q=france"]
```

Comment:
ok but prepare an issue for another PR with a text file listing all slow requests instead of writing it in clear in this array.

Reply:
Yes, I have it in my backlog.
Just wanted a first version in production first.
Trying to get into the build, build, build mindset, you know ;)